docs/Makefile

```diff
@@ -2,7 +2,7 @@
 #
 
 # You can set these variables from the command line.
-SPHINXOPTS    = -W
+SPHINXOPTS    = -j auto -W
```

Comment:
This probably should explain in the commit message details on the upstream bug being fixed in the previous commit.